school.rb

```diff
@@ -0,0 +1,15 @@
+class School
+
+	attr_reader :age
+	attr_accessor :name
+
+	def initialize(options={})
+		@age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
+  end
```

Comment:
I know indentation doesn't always seem important, but if your code is poorly indented in a job setting, your code will not be accepted.  It also makes you look unprofessional.  Please make an effort to follow style guides.